cassiopeia/solver/SpectralNeighborJoiningSolver.py

```diff
@@ -5,32 +5,44 @@
 published in the SIAM Journal for Math and Data Science. 
  
 """
-
+from cassiopeia.data import CassiopeiaTree
+from cassiopeia.solver import (
+    dissimilarity_functions,
+    DistanceSolver,
+    DistanceSolver,
```

Comment:
Duplicate import

Reply:
Once again, the import structure is off here. 

Recall that the structure is -- 

native python libraries (e.g. `os`, `typing`)

third party libraries

cassiopeia modules